File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cs
public class Solution {
    public bool solution(string s) {
        bool answer = true;
        
        if(s.Length != 4 && s.Length != 6){
            answer = false;
        }
        else{
            for(int i = 0; i < s.Length; i++){
                if((int)s[i] < 48 || (int)s[i] > 57){
                    answer = false;
                }
            }
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.py
def solution(s):
    answer = True
    
    length = len(s)
    
    if(length != 4 and length != 6):
        answer = False
    else:
        for item in s:
            if(ord(item) < 48 or ord(item) > 57):
                answer = False
                break;
    
    return answer